examples/internal/proto/examplepb/unannotated_echo_service.swagger.json

```diff
@@ -449,7 +449,8 @@
         "url": "https://github.com/grpc-ecosystem/grpc-gateway"
       },
       "required": [
-        "id"
+        "id",
+        "num"
```

Comment:
This was a bug where num was marked as `required` but didn't show up in the object level array